README.md

```diff
@@ -1,5 +1,23 @@
-# Morpho Blue IRM
+# Morpho Blue IRMs
+
+Some interest rate models for Morpho Blue:
+
+- [AdaptiveCurveIRM](./src/AdaptiveCurveIrm.sol)
+
+## Resources
+
+- AdaptiveCurveIRM: [documentation](https://www.notion.so/morpho-labs/Morpho-Blue-Documentation-Hub-External-00ff8194791045deb522821be46abbdc?pvs=4#d8269074bfd649009f28625a9caa38ea), [announcement article](https://morpho.mirror.xyz/aaUjIF85aIi5RT6-pLhVWBzuiCpOb4BV03OYNts2BHQ).
 
 ## Audits
 
-All audits are stored in the [audits](./audits/)' folder.
\ No newline at end of file
+All audits are stored in the [audits](audits)' folder.
+
+## Licences
+
+The primary licence is MIT, see [LICENCE](LICENSE).
+
+## Getting started
```

Comment:
in other repos it comes before the license and audits

Reply:
I prefer it like that but I can change, it's not important

Reply:
Usually in open-source repos we see the license last I believe because "Getting Started" is the interesting part to sell your open-source product
I'm open to change, though ideally we are consistent everywhere

Btw it's "License" not "Licence" right?